### DISCUSSION TITLE
About ITSG's satellite orbit products

### DISCUSSION THREAD

**Discussion by ztz99Asdog** [groops-devs/groops#298] [Q&A] [answered]
Hello developers，

I would like to inquire about the low Earth orbit satellite orbit information within the satelliteOrbitProducts. If the coordinates of these low Earth satellite orbits are converted to ITRF coordinates, which version of ITRF do they correspond to? Alternatively, what version of ITRF GNSS data did you use to calculate these low Earth satellite orbits?

Sincerely.

Zhaoyi Chen

**Comment by tmayerguerr** [comment, marked as answer]
Hi 

Unfortunately, we have to admit that the reference frame of for our solutions is not entirely clearly defined.
Our main application is gravity field determination and the exact frame is not important here. The situation at moment: We estimate our own global network solution with station coordinates aligned to IGS14b/IGS20 (switch was somewhat with beginning of 2023) and Earth rotation parameters. The LEO orbits were determined in the space fixed frame. To express the orbits in IGS20, one would have to consider also our estimated Earth rotation parameters.

We are planning a complete reprocessing of the orbits in the first quarter of next year. As a lesson learned, the reprocessed orbits will also be published in the earth-fixed frame closely aligned to IGS20.

best regards
Torsten

**Reply by ztz99Asdog** [reply]
Thank you so much for your help! I am very much looking forward to your reprocessed orbits.
